From b87ce349ef05e1f429ba5f1d9f66e3c978d32a01 Mon Sep 17 00:00:00 2001
From: Olaf Hering <olaf@aepfle.de>
Date: Sat, 5 Jun 2010 09:45:12 +0200
Subject: [PATCH] powerpc: chrp/time.c fix preprocessor conditional

commit 1f0f6ec523917972127caf33dee552990db1841e in tip.

Use correct preprocessor syntax to fix compile error

arch/powerpc/platforms/chrp/time.c:86:5: error: "CONFIG_PREEMPT_RT" is not defined

Signed-off-by: Olaf Hering <olaf@aepfle.de>
Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
---
 arch/powerpc/platforms/chrp/time.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/powerpc/platforms/chrp/time.c b/arch/powerpc/platforms/chrp/time.c
index 8f1d8cd..c418c83 100644
--- a/arch/powerpc/platforms/chrp/time.c
+++ b/arch/powerpc/platforms/chrp/time.c
@@ -83,7 +83,7 @@ int chrp_set_rtc_time(struct rtc_time *tmarg)
 	unsigned char save_control, save_freq_select;
 	struct rtc_time tm = *tmarg;
 
-#if CONFIG_PREEMPT_RT
+#ifdef CONFIG_PREEMPT_RT
 	if (!spin_trylock(&rtc_lock))
 		return -1;
 #else
-- 
1.7.0.4

